lib/routes.json

```diff
@@ -2453,12 +2523,6 @@
           "required": true,
           "type": "string"
         },
-        "page": {
-          "type": "integer"
-        },
-        "per_page": {
-          "type": "integer"
-        },
```

Comment:
this is incorrect but will be addressed by the GitHub docs team in future. Not very harmful to have that removed for now, a temporary workaround is not worth it.